src/modules/apps/ItemProperties.ts

```diff
@@ -1,7 +1,7 @@
-import { i18n } from '../../init';
-import { FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME, getGame } from '../settings';
-
-export default class ItemProperties extends FormApplication {
+import { i18n } from '../lib/lib';
+import { FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME } from '../settings';
+import { canvas, game } from '../settings';
```

Comment:
- Since `./settings` is not going to be accessed by any other modules, it's probably safe to rename `FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME` to just `MODULE_NAME`.
- Why are `canvas` and `game` exported from `./settings`? These should already be available in the global scope.
- These two lines should be merged.
```suggestion
import { FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME, canvas, game } from './settings';
```

Reply:
- **Since ./settings is not going to be accessed by any other modules, it's probably safe to rename FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME to just MODULE_NAME.** _No problem every FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME reference will go under the "constants" file with the property CONSTANTS.MODULE_NAME this is a standard so it should be ok_
-  **Why are canvas and game exported from ./settings? These should already be available in the global scope.**: _a little trick i use with the typescript for avoid to write this `getGame().modules.get('xxx')` with this trick i can write `game.modules.get('xxx')` (like i was on javascript) this is a limitation of the library https://github.com/League-of-Foundry-Developers/foundry-vtt-types then the gulpfile.js durign the build remove these import and everything is working_
- **These two lines should be merged.** _no need for this after the correction of the first point_